From 339f61e3f9d6efaaf74f4c2f6480890105c531ef Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 22 Jul 2014 14:37:22 -0400
Subject: [virt] kvm/x86: Drop some unused functions from lapic

Message-id: <1406040016-3289-48-git-send-email-pbonzini@redhat.com>
Patchwork-id: 86126
O-Subject: [RHEL7 PATCH v2 047/221] KVM: x86: Drop some unused functions from lapic
Bugzilla: 1116936
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jan Kiszka <jan.kiszka@siemens.com>

Both have no users anymore.

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
Signed-off-by: Gleb Natapov <gleb@redhat.com>
(cherry picked from commit 9576c4cd6b6fa5716400e63618757b76cff6a813)
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kvm/lapic.c b/arch/x86/kvm/lapic.c
index bdee8d0..d86ff15 100644
--- a/arch/x86/kvm/lapic.c
+++ b/arch/x86/kvm/lapic.c
@@ -76,16 +76,6 @@ static inline void apic_set_reg(struct kvm_lapic *apic, int reg_off, u32 val)
  *((u32 *) (apic->regs + reg_off)) = val;
 }
 
-static inline int apic_test_and_set_vector(int vec, void *bitmap)
-{
- return test_and_set_bit(VEC_POS(vec), (bitmap) + REG_POS(vec));
-}
-
-static inline int apic_test_and_clear_vector(int vec, void *bitmap)
-{
- return test_and_clear_bit(VEC_POS(vec), (bitmap) + REG_POS(vec));
-}
-
 static inline int apic_test_vector(int vec, void *bitmap)
 {
  return test_bit(VEC_POS(vec), (bitmap) + REG_POS(vec));
-- 
1.7.1